Springerlink2022update (#42)
springernature/vpat: springerlink.md

<br><br>
Some complex functionality may omit required WAI-ARIA roles, states, or attributes, or misuse WAI-ARIA markup and techniques. We are actively working to find and repair these problems. 
<br><br>

> Multiple landmarks of the same type without unique labels are used on pages rendered in both designs.  

Reviewer: delete this

<br><br>

> Pages found at order.springer.com omit or use inappropriate headings and heading levels, including the use of unsemantic headings for visual effect. Multiple landmarks of the same type without unique labels are also found on these pages. 

Reviewer: ```suggestion
The SpringerLink checkout process pages found at order.springer.com omit or use inappropriate headings and heading levels, including the use of unsemantic headings for visual effect. Multiple landmarks of the same type without unique labels are also found on these pages. 
```

<br><br>

> Pages found at order.springer.com use form fields that rely solely on built-in browser error checking, which can lead to error messages being unavailable for users with older browser technology, or Assistive Technology that doesn't support these features. 

Reviewer: Drop the "pages found at order.springer.com" part and combine with the paragraph above

<br><br>

> Third party scripts may inject content with incorrect or absent information about structure and relationships. These may include content in advertising scripts, in the Freshdesk customer services widget, among others.

Reviewer: ```suggestion
Third party scripts may inject content with incorrect or absent information about structure and relationships. These may include content in advertising scripts and the Freshdesk customer services widget, among others.
```

Video content elsewhere (e.g. chapters or articles) cannot currently be operated with the keyboard. We are actively working to repair this problem. 
<br><br>

> All components on pages located at order.springer.com can be operated by keyboard alone.

Reviewer: delete this - already covered in the first sentence, so no need for a separate mention for the checkout as the issues are now resolved 🎉

</td>
<td>Partially Supports</td>
<td>

> No loss of content or functionality occurs by setting the text spacing properties specified by this criterion anywhere in this product, except for the search results page and homepage  where a collision may occur between some of the filters. 

Reviewer: ```suggestion
No loss of content or functionality occurs by setting the text spacing properties specified by this criterion anywhere in this product, except for the search results page and [homepage](https://link.springer.com) where a collision may occur between some of the filters. 
```

<br><br>
The SpringerLink website contains content that is rendered in two separate designs. We are in the process of transferring all content from the older design to the newer design. 
<br><br>

> The flyout "menu" component in the header of the older design has a fixed position which may prevent users browsing with high magnification from being able to access all items in the menu.  

Reviewer: ```suggestion
The flyout "menu" component in the header of the older design has a fixed position which may prevent users who browse with high magnification from being able to access all items in the menu.  
```

</ul></td>
<td>Partially Supports</td>
<td>
Content in journals, books, series, protocols, reference works or proceedings may include or be completely composed of text in languages other than that defined in the HTML <code>lang</code> attribute. Research content supports the inclusion of correct `lang` attributes for mixed-language content in some areas such as article titles and summary sections, but not all parts of mixed-language pages are covered. We currently have no method of confirming the inclusion of appropriate <code>lang</code> attributes for pieces of text that may use a different language to the main document. 
<br><br>

> Titles in the cart at order.springer.com may be in multiple languages. The correct <code>lang</code> attribute for titles in a language different to that of the overall document is omitted. 

Reviewer: ```suggestion
Titles in the cart at order.springer.com may be published in multiple languages. The correct <code>lang</code> attribute for titles in a language different to that of the overall document is omitted. 
```

<br><br>
When the transfer process is complete, the navigation methods between designs will be broadly consistent. 
<br><br>

>  The checkout configurations deal with sensitive information and have intentionally been made more simple in design, and do not share the same navigation system as the main site. The navigation of these sets of pages is internally consistent. 

Reviewer: ```suggestion
 The checkout configurations deal with sensitive information and have intentionally been made more simple in design. They do not share the same navigation system as the main site, but the navigation of these sets of pages is internally consistent. 
```

<br><br>
When the transfer process is complete, content will use a unified design style with consistent presentation and functionality.
<br><br>

> The checkout configurations deal with sensitive information and have intentionally been made more simple in design, and do not share the same navigation system as the main site. The presentation of these sets of pages is internally consistent, and the tasks performed are unlike those performed elsewhere on the site. 

Reviewer: ```suggestion
The checkout configurations deal with sensitive information and have intentionally been made more simple in design. They do not share the same navigation system as the main site, but the presentation of these sets of pages is internally consistent, and the tasks performed are unlike those performed elsewhere on the site. 
```
